### PR TITLE
Fix some typos and prepare to merge the duplicate codes.

### PR DESCRIPTION
**Pull request by gongweibao**

### CODE REVIEW

File: python/paddle_serving_server/web_service.py [hunk 1]
```diff
@@ -122,7 +122,7 @@ def run_web_service(self):
                               processes=1)
 
     def get_app_instance(self):
-        return self.app_instance
+        return app_instance
```
Comment (guru4elephant) [inline python/paddle_serving_server/web_service.py]: app_instance undefined?

Comment (gongweibao) [reply python/paddle_serving_server/web_service.py]: Done.

**Review by guru4elephant — APPROVED** [review]
LGTM

**Review by guru4elephant — APPROVED** [review]
LGTM

**Pull request merged**